release/apparmor.go

```diff
@@ -133,3 +143,48 @@ func AppArmorFeatures() []string {
 	}
 	return appArmorFeatures
 }
+
+// parser probe related code
+type apparmorParserFeature struct {
+	feature string
+	rule    string
+}
+
+var requestedParserFeatures = []apparmorParserFeature{
+	{"unsafe", "change_profile unsafe /**,"},
+}
+
+// tryParser will run the parser on the rule to determine if the feature is
+// supported.
+func tryParser(rule string) bool {
+	cmd := exec.Command("apparmor_parser", "-p")
```

Comment:
Could you also use the `--preprocess` spelling rather than the short version. This is more discoverable to casual readers.